docs/manual/java/releases/Migration15.md

```diff
@@ -9,21 +9,29 @@ Lagom 1.5 also updates to the latest major versions of Play (2.7) and Akka HTTP
 
 ### Maven
 
-If you're using a `lagom.version` property in the `properties` section of your root `pom.xml`, then simply update that to `1.5.0-M2`. Otherwise, you'll need to go through every place that a Lagom dependency, including plugins, is used, and set the version there.
+If you're using a `lagom.version` property in the `properties` section of your root `pom.xml`, then simply update that to `1.5.0-M3`. Otherwise, you'll need to go through every place that a Lagom dependency, including plugins, is used, and set the version there.
 
 ### sbt
 
 The version of Lagom can be updated by editing the `project/plugins.sbt` file, and updating the version of the Lagom sbt plugin. For example:
 
 ```scala
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.5.0-M2")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.5.0-M3")
 ```
 
 We also recommend upgrading to sbt 1.2.1 or later, by updating the `sbt.version` in `project/build.properties`.
 
 ## Deprecations
 
-TODO
+### Service Ports
+
+In Lagom 1.5.0 we are introducing support for SSL calls for gRPC integration and new build setting is introduced to manually configure https port for a given service.
+
+In maven, the new setting is called `serviceHttpsPost`.  In order to keep the names aligned, we are deprecating  `servicePort` in favour of `serviceHttpPort`.
```

Comment:
`In order to keep the names... ` -> `To keep the names...`.

Same for the paragraph below.